Misc/NEWS.d/next/Library/2024-04-24-07-45-08.gh-issue-118218.m1OHbN.rst

```diff
@@ -0,0 +1 @@
+Speed up :func:`itertools.pairwise` by up to 1.8x
```

Comment:
Perhaps add "in common use case"? Because it makes no difference if you save all result.